Makefile

```
@@ -0,0 +1,43 @@
+# Basic Makefile template for paper-template project
+# For more on Makefiles: https://www.gnu.org/software/make/manual/make.html
+
+# Phony targets (targets that don't represent files)
+.PHONY: all clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)